Sorting/TSorting.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Sorting
+{
+    abstract class TSorting
+    {
+        public abstract int[] Sort();
+
```

Comment:
you should have one more variable that means how you sort array: by ascending or by descending
This was stated in condition
Also you forgot to implement function in which you sum all elements in the array
Please, read the contition one more time and implement things that you didn't do